adafruit_vl53l1x.py

```diff
@@ -301,3 +302,13 @@ def _read_register(self, address, length=1):
             i2c.write(struct.pack(">H", address))
             i2c.readinto(data)
         return data
+
+    def set_address(self, new_address):
+        """
+        Set a new I2C address to the instantaited object. This is only called when using
+        multiple VL53L0X sensors on the same I2C bus (SDA & SCL pins). See also the
+        `example <examples.html#multiple-vl53l1x-on-same-i2c-bus>`_ for proper usage.
+        """
+        new_addres_as_byte_string = new_address.to_bytes(1, "little")
```

Comment:
this is kinda wierd way of making a byte string, can you not use `bytes()`? check our other libraries for ideas

Reply:
How about struct.pack()?